lib/pages/about.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:capstone_app/main.dart';
+import 'package:flutter/material.dart';
+import 'package:capstone_app/style/appTheme.dart';
+import 'package:capstone_app/services/auth.dart';
+
+
+class AboutPage extends StatelessWidget {
+  GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
```

Comment:
The navbar drawer icon should be white, the home page has an example of how to do this